services/src/libs/sentry.js

```diff
@@ -36,10 +36,9 @@ class Sentry {
       level: data.log.level.toLowerCase(),
       logger: data.log.name,
       user: {
-        uuid: data.uuid,
+        id: data.uuid,
       },
       extra: {
-        id: data.uuid,
```

Comment:
I added this to, I thought, enable that capture back then. So now I'm removing it to avoid keeping it for naught.